pinot-servicemanager/build.gradle.kts

```diff
@@ -1,9 +1,19 @@
 plugins {
+  `java-library`
   id("org.hypertrace.docker-publish-plugin")
 }
 
 hypertraceDocker {
   defaultImage {
     imageName.set("pinot-servicemanager")
+    dockerFile.set(file("Dockerfile"))
   }
 }
+
+dependencies {
+  implementation("org.apache.pinot:pinot-spi:0.5.0")
+  implementation("org.apache.pinot:pinot-avro-base:0.5.0")
+  implementation("org.hypertrace.core.kafkastreams.framework:kafka-streams-serdes:0.1.11")
```

Comment:
This won't be available at runtime in the current code. Also, I suspect any of our framework libraries bring in a huge dependency tree which will potentially clash with pinot's.

Reply:
we only need classes from - https://dl.bintray.com/hypertrace/maven/org/hypertrace/core/kafkastreams/framework/kafka-streams-serdes/0.1.11/kafka-streams-serdes-0.1.11.jar, rest will convert to compile only. Will try out the way you did it in #33